src/main/java/com/socialbox/service/impl/UserServiceImpl.java

```diff
@@ -21,7 +22,8 @@ public UserServiceImpl(UserRepository userRepository) {
   }
 
   public User getUserById(String id) {
-    return this.userRepository.findById(id).get();
+    Optional<User> userOptional = this.userRepository.findById(id);
+    return userOptional.orElse(null);
```

Comment:
This is how we should handle optionals, if the main goal is to just return them.